slack_bolt/app/app.py

```diff
@@ -335,6 +336,34 @@ def middleware(self, *args) -> None:
                     CustomMiddleware(app_name=self.name, func=middleware_or_callable)
                 )
 
+    # -------------------------
+    # Workflows: Steps from Apps
+
+    def step(
+        self,
+        callback_id: Union[str, Pattern, WorkflowStep],
```

Comment:
So, if you remove the kwargs pattern here, this method can accept only a `WorkflowStep` object as an argument.